spec/spec_helper.rb

```diff
@@ -30,8 +30,16 @@
   SimpleCov.root(File.expand_path('../..', __FILE__))
   SimpleCov.formatter = Coveralls::SimpleCov::Formatter
   SimpleCov.start('rails') do
+    add_filter '/.internal_test_app'
+    add_filter '/bin'
+    add_filter '/db'
+    add_filter '/config'
+    add_filter '/jetty'
+    add_filter '/lib/generators'
+    add_filter '/public'
+    add_filter '/solr_conf'
```

Comment:
I'm not sure we should filter solr_conf, public, jetty, db, bin, vendor, because we don't have any ruby files in there anyway.